vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -150,10 +176,49 @@ func newRefCount() *refCount {
 	}
 }
 
-// Init Refcounts. Discover volume usage refcounts from Docker.
-// This functions does not sync with mount/unmount handlers and should be called
-// and completed BEFORE we start accepting Mount/unmount requests.
-func (r RefCountsMap) Init(d drivers.VolumeDriver, mountDir string, name string) {
+// return if refcounting has been successfull
+func (r *RefCountsMap) GetSuccess() bool {
+	return r.refcntSuccess
+}
+
+// dirty the background refcount process
+func (r *RefCountsMap) Dirty() {
```

Comment:
Nit: can we use a verb or verb phrase to name a function? For this function, we may rename it to markDirty().